src/lenskit/metrics/reranking/_rbo.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of LensKit.
+# Copyright (C) 2018-2023 Boise State University.
+# Copyright (C) 2023-2025 Drexel University.
+# Licensed under the MIT license, see LICENSE.md for details.
+# SPDX-License-Identifier: MIT
+
+
+from __future__ import annotations
+
+import numpy as np
+
+from lenskit.data import ItemList
+from lenskit.metrics import GeometricRankWeight
+
+
+def RBO(reference: ItemList, reranked: ItemList, p: float = 0.85, k: int = 10) -> float:
```

Comment:
Since these are functions, let's name them lower-case, possibly snake-case, to save the uppercase names for future metric classes.

Possibly `rank_biased_overlap` and `least_item_promoted`? No harm in a full-length name.